src/main/java/uk/gov/companieshouse/web/accounts/controller/smallfull/AddOrRemoveLoansController.java

```diff
@@ -100,6 +100,10 @@ public String getAddOrRemoveLoans(@PathVariable String companyNumber,
                 addOrRemoveLoans.getLoanToAdd().setDirectorName(validDirectorNames.get(0));
             }
 
+            if(!validDirectorNames.isEmpty()) {
+                validDirectorNames.add("Prefer not to say");
```

Comment:
This string should probably be a constant

Reply:
Changed the string to constant